src/vivarium_public_health/population/mortality.py

```diff
@@ -83,13 +101,69 @@ def _get_mortality_rate(self, builder: Builder) -> Pipeline:
             requires_columns=["age", "sex"],
         )
 
+    # noinspection PyMethodMayBeStatic
+    def _get_all_cause_mortality_rate(self, builder: Builder) -> LookupTable:
+        acmr_data = builder.data.load("cause.all_causes.cause_specific_mortality_rate")
+        return builder.lookup.build_table(
+            acmr_data, key_columns=["sex"], parameter_columns=["age", "year"]
+        )
+
     # noinspection PyMethodMayBeStatic
     def _get_life_expectancy(self, builder: Builder) -> LookupTable:
         life_expectancy_data = builder.data.load(
             "population.theoretical_minimum_risk_life_expectancy"
         )
         return builder.lookup.build_table(life_expectancy_data, parameter_columns=["age"])
 
+    # noinspection PyMethodMayBeStatic
+    def _get_raw_unmodeled_csmr(self, builder: Builder) -> LookupTable:
```

Comment:
Why not just put all this in the artifact under an "unmodeled_causes" key?

Reply:
I like that this makes it explicit in the config file which causes are included in the "unmodeled causes"